doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -838,6 +841,12 @@ Modem libraries
 
     * The RTT trace backend to allocate the RTT channel at boot, instead of when the modem is activated.
     * The flash trace backend to solve concurrency issues when reading traces while writing, and when reinitializing the application (warm start).
+    * Renamed the nRF91 socket offload layer from ``nrf91_sockets`` to ``nrf9x_sockets`` to reflect that the offload layer is not exclusive to the nRF91 Series SiPs.
+    * The :ref:`modem_trace_module` to let the trace thread sleep when the modem trace level is set to :c:enum:`NRF_MODEM_LIB_TRACE_LEVEL_OFF` using the :c:func:`nrf_modem_lib_trace_level_set` function, and the trace backend returns ``-ENOSPC``.
```

Comment:
FYI, `enum` is for the type, `enumerator` is for the value.

```suggestion
    * The :ref:`modem_trace_module` to let the trace thread sleep when the modem trace level is set to :c:enumerator:`NRF_MODEM_LIB_TRACE_LEVEL_OFF` using the :c:func:`nrf_modem_lib_trace_level_set` function, and the trace backend returns ``-ENOSPC``.
```

Reply:
`:c:enum:` is used elsewhere.

Reply:
should be :c:enumerator: . Same is used here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16847/nrf/_sources/libraries/modem/nrf_modem_lib/nrf_modem_lib_trace.rst.txt.

Reply:
See https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#doxybridge.

Reply:
I did  a quick of "NRF_MODEM_LIB_TRACE_LEVEL_OFF" in the NCS docs, and we use :c:enumerator: for this API. Couldn't find :c:enum: used for "NRF_MODEM_LIB_TRACE_LEVEL_OFF" in the docs. 
My bad, missed this in the initial review.